rush.json

```diff
@@ -212,7 +212,8 @@
          * your PR branch, and in this situation "rush change" will also automatically invoke "git fetch"
          * to retrieve the latest activity for the remote master branch.
          */
-        "url": "git@github.com:gooddata/gooddata-ui-sdk.git"
+        "url": "git@github.com:gooddata/gooddata-ui-sdk.git",
+        "defaultBranch": "master" // branch used when "rush change" is called (current rush default value is "master")
```

Comment:
Why is this needed when it's same as the default value?

Reply:
Good catch. The current value is `main`. I updated the comment. I was rewriting it few times and screw it up in the end anyway 😅